.github/workflows/run-demo.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Install gittuf
         uses: gittuf/gittuf-installer@f31e69c7c18c7473cbce18ed69a559b945d3a738
         with:
-          gittuf-version: 0.8.0
+          gittuf-version: 0.9.0
```

Comment:
Can you update this to drop the version? Let's just use the default so a dependabot update for gittuf-installer is sufficient.